docs/CM_Tutorial.adoc

```diff
@@ -239,19 +239,20 @@ ChronicleSet<UUID> uuids =
         .create();
 ----
 
+[#custom-serializers]
 == Custom serializers
 
-Chronicle Map allows you to configure custom marshallers for key or value types which are not supported out-of-the-box. You can also serialize supported types like `String` in some custom way (encoded other than UTF-8), or serialize supported types more efficiently than by default.
+Chronicle Map allows you to configure custom marshallers for key or value types which are not supported out-of-the-box.You can also serialize supported types like `String` in some custom way (encoded other than UTF-8), or serialize supported types more efficiently than by default.
 
-There are three pairs of serialization interfaces. Only one of them should be chosen in a single implementation, and supplied to the `ChronicleMapBuilder` for the key or value type. These are:
+There are three pairs of serialization interfaces.Only one of them should be chosen in a single implementation, and supplied to the `ChronicleMapBuilder` for the key or value type.These are:
```

Comment:
why remove space after "."? 

This makes the text harder to read and edit

Reply:
Not sure why this happened. I will revert.